dist/chart/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
Any idea why we have dependencies in `Chart.lock` without them being specified in `Chart.yaml`?